roles/nf-core-biocontainer-temp/tasks/main.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: install nf-core/rnaseq
+  command: "{{ nf_core_env }}/bin/nf-core -v download {{ item.name }}
+   --compress none --outdir {{ sw_path }}{{ item.name }} --release {{ item.release }}"
+  environment:
+    PATH: "{{ nf_core_env }}/bin:{{ ansible_env.PATH }}"
+  args:
+    chdir: "{{ sw_path }}"
+  with_items: "{{ pipelines }}"
+
+- name: Create directory for rnaseq biocontainers
+  file:
+    path: "{{ ngi_containers }}/biocontainers"
+    state: directory
+    mode: 02775 #ug=rwx, o=rx, g+s
+
+- name: Create temp directories for download
+  file:
+    path: "{{ ngi_containers }}/download"
+    state: directory
+
+#Download these for rnaseq 3 containers until nf-core dowload is updated
+#wget -L https://dl.dropboxusercontent.com/s/5venz3l2wihaqdp/nf-core-rnaseq_v3.0.tar.gz
+#wget -L https://dl.dropboxusercontent.com/s/0w2e72pz2iuj37e/nf-core-rnaseq_v3.0.tar.gz.md5
```

Comment:
We don't download the checksum, so maybe skip the comment about it.

Reply:
Done!